src/service/video/docking.js

```diff
@@ -600,11 +600,12 @@ export class VideoDocking {
 
   /**
    * Reconciliates the state of a docked or potentially dockable video when
-   * its position changes.
+   * something (viewport, position) changes.
    * @param {!../../video-interface.VideoOrBaseElementDef} video
+   * @param {boolean=} onPositionChange
    * @private
    */
-  onPositionChange_(video) {
+  update_(video, onPositionChange = false) {
```

Comment:
Using flag arguments is a little bit of an antipattern. Consider adding a separate function for the `onPositionChange == true` case.

https://martinfowler.com/bliki/FlagArgument.html

Reply:
Done.